CHANGELOG.md

```diff
@@ -104,6 +104,10 @@
   [Oleg Kokhtenko](https://github.com/kohtenko)
   [#5405](https://github.com/realm/SwiftLint/issues/issue_number)
 
+* Add `excluded_patterns` to `line_length` to avoid linting lines
+  that match the patterns.
```

Comment:
Please append two spaces to the end of the description. They are required to enforce a line break in the rendered Markdown.


You say here that the patterns must **match** the line but the implementation checks for **containment**.